pywhat/printer.py

```diff
@@ -87,9 +87,12 @@ def print_json(self, text: dict):
 
     def get_crypto_links(self, text, matched):
         explorers = {
-            "Ethereum Wallet": "https://etherscan.io/address/",
-            "Dogecoin Wallet Address": "https://dogechain.info/address/",
-            "Bitcoin Wallet": "https://www.blockchain.com/btc/address/",
+            "Ethereum (ETH) Wallet Address": "https://etherscan.io/address/",
+            "Dogecoin (DOGE) Wallet Address": "https://dogechain.info/address/",
+            "Bitcoin (BTC) Wallet Address": "https://www.blockchain.com/btc/address/",

```

Comment:
Also this is annoying -- but can we test that these links show up? You can do so by:
This is annoying, but can you also add all the tests to:
https://github.com/bee-san/pyWhat/blob/main/fixtures/file

And copy / paste into:
https://github.com/bee-san/pyWhat/blob/main/tests/test_click.py

and then in  that Click file trying to match the dogechain / etherscan URL etc :-) (I'd just do `re.searchall("dogechain")` to find it 😄